.github/workflows/build.yml

```diff
@@ -6,6 +6,8 @@ jobs:
   build:
     name: Build for Node version ${{ matrix.node-version }}
     runs-on: ubuntu-22.04
+    env:
+      FORCE_COLOR: 1
```

Comment:
For some reason, the underlying `colors` dependency of `json-diff` assumes colors aren't supported in the CI. I tried forcing colors programmatically with the `{ color: true }` option that `json-diff` supports but it didn't work. Using this environment variable makes it work though.
We need that because the test for `warn-upload-overwrite` has assertions on the diff displayed to the end user